internal/services/postgres/validate/flexible_server_storage_tier_test.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
I assume that we don't need it anymore if using "IsExplicitlyNullInConfig" to fix the issue.